src/ui/src/js/configs/dt_renderer.js

```diff
@@ -26,42 +26,42 @@ export default function runDTRenderer(DTRendererService) {
             $('<label>')
               .attr('for', 'childCheck')
               .css('padding-left', '4px')
-              .text('Include Children')
+              .text('Show Children')
           );
         $('.dataTables_filter').prepend(childContainer);
       }
 
       if (options && options.hiddenRequestContainer) {
-          let hiddenRequestContainer = $('<span>')
-            .attr('id', 'hiddenRequestContainer')
-            .css('margin-right', '20px')
-            .append(
-              $('<input>')
-                .attr('id', 'hiddenRequestCheck')
-                .attr('type', 'checkbox')
-                .css('margin-top', '-4px')
-                .change(() => { $('.dataTable').dataTable().fnUpdate(); })
-            )
-            .append(
-              $('<label>')
-                .attr('for', 'hiddenRequestCheck')
-                .css('padding-left', '4px')
-                .text('Include Hidden')
-            );
-          $('.dataTables_filter').prepend(hiddenRequestContainer);
-        }
+        let hiddenRequestContainer = $('<span>')
+          .attr('id', 'hiddenRequestContainer')
+          .css('margin-right', '20px')
+          .append(
+            $('<input>')
+              .attr('id', 'hiddenRequestCheck')
+              .attr('type', 'checkbox')
+              .css('margin-top', '-4px')
+              .change(() => { $('.dataTable').dataTable().fnUpdate(); })
+          )
+          .append(
+            $('<label>')
+              .attr('for', 'hiddenRequestCheck')
+              .css('padding-left', '4px')
+              .text('Show Hidden')
+          );
+        $('.dataTables_filter').prepend(hiddenRequestContainer);
+      }
 
       if (options && options.hiddenContainer) {
         let hiddenContainer = $('<ul>')
-              .attr('id', 'hiddenList')
-              .css('margin-right', '20px')
-              .attr('style', 'list-style-type:none;margin-right:20px; display:inline; padding-left:0px;')
-              .append(
-                  $('<label>')
-                  .attr('for', 'childCheck')
-                  .css('padding-left', '4px')
-                  .text('Show Hidden')
-              )
+          .attr('id', 'hiddenList')
+          .css('margin-right', '20px')
+          .attr('style', 'list-style-type:none;margin-right:20px; display:inline; padding-left:0px;')
+          .append(
+            $('<label>')
+            .attr('for', 'filterHidden')
```

Comment:
If you are going to change the attribute name, you also need to update request_index.js line 43 because it references this attribute.

Reply:
This is ... confusing. I really wish there were a better way to do this. I even spent some time trying to rework this but failed :/

The request index page doesn't use this chunk, it uses the chunk above (`if (options && options.hiddenRequestContainer)`). This one is only used on the command index page.